### PR TITLE
CRL-1646 - Implemented : Update Detection for abnormally_high_instanc…

### PR DESCRIPTION
**Pull request by jcbrewer5D6400**
…es_launched to use MLTK

### CODE REVIEW

File: baselines/train_ec2_excessive_runinstances.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+baseline:
+  splunk:
+    macros:
+      - ec2_excessive_runinstances_mltk_input_filter
+    schedule:
+      cron_schedule: ''
+      earliest_time: -91d@d
+      latest_time: -1d@d
+    search: sourcetype=aws:cloudtrail eventName=RunInstances errorCode=success | bucket span=10m _time | stats sum(instances_launched) as instances_launched by _time userName | fit DensityFunction instances_launched threshold=0.0005 into ec2_excessive_runinstances_v1
```
Comment (patel-bhavin) [inline baselines/train_ec2_excessive_runinstances.yml]: We need the search syntax to use datamodels and have the macros in it. Similar to 
https://github.com/splunk/security-content/blob/73800355367d8b99ff69a9596b908f3bdda76e52/baselines/baseline_cloud_compute_regions.yml#L11

File: detections/ec2_excessive_runinstances_mltk.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+data_metadata:
+  data_source:
+    - AWS CloudTrail logs
+  data_sourcetypes:
+    - aws:cloudtrail
+  providing_technologies:
+    - AWS
+description: This search looks for CloudTrail events where a user successfully launches
+  an abnormally high number of instances.
+detect:
```
Comment (patel-bhavin) [inline detections/ec2_excessive_runinstances_mltk.yml]: this manifest files needs a "baseline" key. Similar to 

https://github.com/splunk/security-content/blob/73800355367d8b99ff69a9596b908f3bdda76e52/detections/cloud_compute_activity_in_new_region.yml#L2

File: detections/ec2_excessive_runinstances_mltk.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      risk:
+        risk_object: userName
+        risk_object_type:
+          - user
+        risk_score: 30
+      schedule:
+        cron_schedule: '*/10 * * * *'
+        earliest_time: -70m@m
+        latest_time: -10m@m
+      search: sourcetype=aws:cloudtrail eventName=RunInstances errorCode=success | bucket span=10m _time | stats sum(instances_launched) as instances_launched by _time userName | apply ec2_excessive_runinstances_v1 | rename "IsOutlier(instances_launched)" as isOutlier | where isOutlier=1
```
Comment (patel-bhavin) [inline detections/ec2_excessive_runinstances_mltk.yml]: use datamodels and embed macros

File: detections/ec2_excessive_runinstances_mltk.yml (hunk 2 above)
Comment (jcbrewer5D6400) [reply detections/ec2_excessive_runinstances_mltk.yml]: Done in my last commit.

File: baselines/train_ec2_excessive_runinstances.yml (hunk 1 above)
Comment (jcbrewer5D6400) [reply baselines/train_ec2_excessive_runinstances.yml]: I've added the input filter macro in my last commit. Dorsey mentioned that we could do a separate AWS and generic cloud analytics stories for this.

File: detections/ec2_excessive_runinstances_mltk.yml (hunk 3 above)
Comment (jcbrewer5D6400) [reply detections/ec2_excessive_runinstances_mltk.yml]: I've added the input filter macro in my last commit. Dorsey mentioned that we could do a separate AWS and generic cloud analytics stories for this.

File: detections/ec2_excessive_runinstances_mltk.yml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    - CIS 13
+  kill_chain_phases:
+    - Actions on Objectives
+  mitre_attack:
+    - Execution
+  nist:
+    - DE.DP
+    - DE.AE
+modification_date: '2019-11-14'
+name: Abnormally High AWS Instances Launched by User
```
Comment (patel-bhavin) [inline detections/ec2_excessive_runinstances_mltk.yml]: I think we should specify MLTK in the search name

File: stories/cloud_cryptomining.yml [hunk 5]
```diff
@@ -13,7 +13,7 @@ detections:
   - detection_id: c6ddbf53-9715-49f3-bb4c-fb2e8a309cda
     name: Cloud Compute Instance Created With Previously Unseen Instance Type
     type: splunk
-  - detection_id: 8dbcf362-ae43-44ea-9084-77db4bed2c68
+  - detection_id: bc24922d-987c-4645-b288-f8c73ec194c4
```
Comment (patel-bhavin) [inline stories/cloud_cryptomining.yml]: not sure why this id changed. Also, add your MLTK detection object in this story

Comment (jcbrewer5D6400) [reply stories/cloud_cryptomining.yml]: I changed it b/c the detection_id referenced doesn't exist. The one I changed it to is the correct one accd to my searches and a brief chat with @trogdorsey  last week.

Comment (patel-bhavin) [reply stories/cloud_cryptomining.yml]: right on! are we adding your MLTK detection to this story?

Comment (jcbrewer5D6400) [reply stories/cloud_cryptomining.yml]: Yes. I made the commit just now.

**Pull request merged**